tests/integration/index/create_test.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package index
```

Comment:
todo: Sorry if I'm jumping the gun a bit and you are already working/planning on this, but I think we also need tests to assert that `GetIndexes` actually returns the indexes created.